students/ltrisolini/Lesson05/mailroom3.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import sys
+from collections import defaultdict
+
+donors = defaultdict(list, {'Andy': [10.00], 'Bill': [15.00, 25.00], 'Chuck': [20.00, 30.00, 40.00]})
+
+def menu_selection(prompt, dispatch_dict):
+
+    while True:
+        try:
+            response = input(prompt)
+            dispatch_dict[response]()
+        except KeyError:
```

Comment:
it is best to only have try/except around code that will throw; so only around line 13